train_model.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+import time
+import argparse
+from sklearn import linear_model
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.neighbors import KNeighborsClassifier
+import numpy as np
+from termcolor import colored
+import pickle
+
+def parse(line):
+    tokens = line.split(',')
+    features = [tokens[3], tokens[4], tokens[5], tokens[13], tokens[14], tokens[15], tokens[16], 
+        tokens[17], tokens[20], tokens[21], tokens[22]]
+    features = [float (f) for f in features]
+    label = int (tokens[23])
+    return (features, label)
+
+start_time = time.time()
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--model", choices=["random_forest", "k_neighbors", "logistic_regression"], default=["random_forest"], dest="model")
+parser.add_argument("--train_dataset_size", default=100000, type=int, dest="train_dataset_size")
+args = parser.parse_args()
+print "Model used in this run is %s" % colored(args.model, "red")
+print "Train dataset size is %s" % colored(args.train_dataset_size, "red")
+
+train = open ('train.csv', 'r')
```

Comment:
Please wrap L28-L41 into a function 
`def load_dataset_from_file(filename, examples_count):`
and move it to the `common` module.